src/components/FeeMethodSelector.tsx

```diff
@@ -164,51 +187,62 @@ function FeeMethodSelector() {
     chooseAcalaRelayer,
```

Comment:
React Hook useEffect has missing dependencies: 'chooseNeonRelayer', 'neonRelayerEligible', and 'targetIsNeon'.

Reply:
It useEffect hook dependences was updated